gtsfm/runner/gtsfm_runner_base.py

```diff
@@ -127,6 +127,25 @@ def construct_argparser(self) -> argparse.ArgumentParser:
             default=None,
             help="tmp directory for dask workers, uses dask's default (/tmp) if not set",
         )
+        parser.add_argument(
+            "--run_cluster",
+            action="store_true",
+            default=False,
+            help="whether to run gtsfm on a cluster"
+        )
+        parser.add_argument(
+            "--cluster_config_name",
```

Comment:
just `--cluster_config` should do, technically its `--cluster_config_path`, but lets keep it short ?